jobqueue/src/main/java/com/path/android/jobqueue/network/NetworkUtilImpl.java

```diff
@@ -36,7 +38,16 @@ public boolean isConnected(Context context) {
             return false;
         }
 
+        int permState = context.checkCallingOrSelfPermission(Manifest.permission.ACCESS_NETWORK_STATE);
+        if (permState != PackageManager.PERMISSION_GRANTED) {
```

Comment:
i need to double check but i think this permission is auto-granted when present in the manifest.
If it is auto-granted, when not present, I'd rather the app to crash (which will happen during development) so that developer knows that they should add the permission. 
Not doing so creates hidden issues that developer never recognizes.